pom.xml

```diff
@@ -551,6 +551,15 @@
 			<artifactId>mail</artifactId>
 			<version>1.4.5</version>
 		</dependency>
+		
```

Comment:
Please ignore any local changes I have made. The only one which matters is this solr dependency.